graph/boruvka.cpp

```diff
@@ -0,0 +1,191 @@
+/**
+ * @file boruvkas_algorithm.cpp
+ * @brief Implementation of Boruvka's Algorithm to find out the minimum spanning
+ * tree of a graph.
+ * @author FranciscoThiesen
+ * @details For more details, see
+ * https://en.wikipedia.org/wiki/Bor%C5%AFvka%27s_algorithm
+ */
+
+#include <cassert>
+#include <iostream>
+#include <numeric>
+#include <vector>
```

Comment:
The purpose of these headers must be documented
A simple comment would suffice.

```cpp
#include <header> /// for <reason>
```

Reply:
Done!